deployments/Dockerfiles/tendermint/app.py

```diff
@@ -82,6 +85,48 @@ def override_config_toml() -> None:
         fp.write(config)
 
 
+def update_peers(validators: List[Dict]) -> None:
+    """Fix peers."""
+
+    config_path = Path(os.environ["TMHOME"]) / "config" / "config.toml"
+    config_text = config_path.read_text(encoding="utf-8")
+
+    new_peer_string = 'persistent_peers = "'
+    for peer in validators:
+        hostname = peer["hostname"]
+        if hostname in ("localhost", "0.0.0.0"):
+            hostname = DOCKER_INTERNAL_HOST
```

Comment:
This node will be running in a docker container and no other node will be running in the same container, if. we receive either `localhost` or `0.0.0.0`, we make an assumption that the address belongs to a node running on the same machine with a different docker container and different p2p port so we replace the hostname with the docker's internal host url

Reply:
This should be a comment in code!